src/patternfly/components/Menu/menu.scss

```diff
@@ -76,6 +82,22 @@
   --pf-c-menu__item-action--m-favorite__icon--FontSize: var(--pf-global--icon--FontSize--sm);
   --pf-c-menu__item-action--m-favorite--m-favorited__icon--Color: var(--pf-global--palette--gold-400);
 
+  // Drilldown
+  --pf-c-menu--m-drilldown--Width: 250px;
```

Comment:
do we need to define this? The menu will be absolutely positioned so it won't be full-width when it's used. Can we just let it grow like a regular dropdown menu, and have this set to `auto` by default, so that you can override the var if you want to set a width? I would be in favor of setting `--pf-c-menu--Width/Height` just on the parent component. There is a `maxHeight` prop on the react select component that sets `max-height: {maxHeight}; overflow: auto;`. I've argued that should be a core modifier that sets a reasonable max-height by default, then is overridable via `--pf-c-menu--Height`.